test_app/tests/lib/routers/test_association_resoure_router.py

```diff
@@ -263,3 +265,20 @@ def test_related_custom_actions_get_scrubed(organization, method, admin_api_clie
     else:
         response = admin_api_client.get(url)
     assert response.status_code == 404
+
+
+def test_autogen_viewset_attributes():
+    sublist_viewset = None
+    for url, viewset, view_name in router.registry:
```

Comment:
@john-westcott-iv fun that this just came up in our meeting this morning. Why not add the auto-generated viewset classes into a registry? Because they are _already in a registry_. It's probably a little non-obvious how to get them out, but hey, that's development.

Reply:
On initialization of the `router` we could save a reference to its registry in the association_resource_router module. You know, so we can have a registry for your registries. So you can be sure you're registered while you register.